src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -355,27 +354,46 @@ public boolean isNativeMethod() {
      * @revised 9
      * @see    Throwable#printStackTrace()
      */
+    @Override
```

Comment:
The override makes no difference.

Reply:
ack. I can remove if necessary, but [stylistically prefer to mark overridden methods as such](https://google.github.io/styleguide/javaguide.html#s6.1-override-annotation) (and automatically enforce via [static analysis](https://errorprone.info/bugpattern/MissingOverride) )